### PR TITLE
Include specs in var-meta

### PR DESCRIPTION
**Pull request by dpsutton**
Wasn't threaded before so the value was computed and thrown away.

Before submitting a PR make sure the following things have been done:

- [x] The commits are consistent with our [contribution guidelines](CONTRIBUTING.md)
- [x] You've added tests to cover your change(s)
- [x] All tests are passing
- [x] The new code is not generating reflection warnings

Keep in mind that new orchard builds are automatically deployed to Clojars
once a PR is merged, but **only** if the CI build is green.

Thanks!

### CODE REVIEW

File: test/orchard/meta_test.clj [hunk 1]
```diff
@@ -2,7 +2,9 @@
   (:require
    [orchard.meta :as m]
    [clojure.repl :as repl]
-   [clojure.test :refer :all]))
+   [clojure.test :refer :all]
+   [clojure.set :as set]
+   [clojure.spec.alpha :as s]))
```
Comment (bbatsov) [inline test/orchard/meta_test.clj]: The spec code should be conditionally added, as we still support Clojure 1.8.

Comment (dpsutton) [reply test/orchard/meta_test.clj]: yeah just saw the test matrix failure. I'll get to this tonight.

**Comment by dpsutton** [comment]
@bbatsov good to go

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**